src/app/core/services/external-machine-deployment.ts

```diff
@@ -132,8 +132,8 @@ export class ExternalMachineDeploymentService {
   ): Observable<boolean> {
     const dialogConfig: MatDialogConfig = {
       data: {
-        title: 'Delete Machine Deployment',
-        message: `Delete <b>${md.name}</b> machine deployment of <b>${cluster.name}</b> cluster permanently?`,
+        title: `Delete ${cluster?.cloud.eks ? 'Node Group' : 'Node Pool'}`,
```

Comment:
do we need to tell the user that the node will be remove from the cloud provider’s end?